### DISCUSSION TITLE
esp 32 s3 7 inch new type display cant able to flash klipper

### DISCUSSION THREAD

**Discussion by DMAG3D** [suchmememanyskill/CYD-Klipper#111] [General]
https://www.waveshare.com/esp32-s3-touch-lcd-7.htm
this display im using but cant able to flash klipper ui using yours or https://github.com/suchmememanyskill/CYD-Klipper,
cant able to get hands on cyd screen ,only got this in hands

**Comment by suchmememanyskill** [comment]
Cyd-Klipper is not compatible with this display nor has any infrastructure to support this display

**Reply by DMAG3D** [reply]
CAN YOU GIVE LIST OR LINKS OF ALL current supported displays above 2 inchessize as we cant easily have hands on chinese diaplay in our country

**Reply by suchmememanyskill** [reply]
@DMAG3D ![Screenshot_20240627-104522_Chrome_1.png](https://github.com/suchmememanyskill/CYD-Klipper/assets/38142618/f35138ad-59f8-4f4c-a306-275d5906ce7b)

**Reply by DMAG3D** [reply]
https://www.amazon.in/DIYmall-ESP32-3248S035C-Dual-core-320x480-Capacitive/dp/B0C5DB6RHM will this supports

**Reply by suchmememanyskill** [reply]
Looks right

**Reply by DMAG3D** [reply]
i was contineously sourcing on alibaba and aliexpress as well ,since morning but still cant find any link of 4.3 inch display module,does 4.3/7 inch is not available anywhere thats mentioned in your above image, if available than pls attach links
